packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -41,28 +41,46 @@ function composeObjectExpressionPropertyIntoString(
   return composedProperty;
 }
 
+function parseLoadStringArgument(argument: string): string[] {
+  let properties: string[] = [];
+  argument
+    .replace(/\s/g, "")
+    .split(",")
+    .forEach((property: string) => {
+      properties.push(property);
+    });
+
+  return properties;
+}
+
 export function parseLoadArguments(node: TSESTree.MemberExpression): string[] {
   node = findTopLevelExpression(node);
 
   if (
     isLoadFunction(node) &&
     node.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression
   ) {
+    const argument = node.parent.arguments[0];
+    if (!argument) return [];
+
     let properties: string[] = [];
-    node.parent.arguments.forEach(
-      (argument: TSESTree.CallExpressionArgument) => {
-        if (argument.type === TSESTree.AST_NODE_TYPES.Literal) {
-          (argument.value as string)
-            .replace(/\s/g, "")
-            .split(",")
-            .forEach((property: string) => {
-              properties.push(property);
-            });
-        } else if (argument.type === TSESTree.AST_NODE_TYPES.ObjectExpression) {
-          properties.push(composeObjectExpressionPropertyIntoString(argument));
+    if (argument.type === AST_NODE_TYPES.ArrayExpression) {
+      argument.elements.forEach((args) => {
```

Comment:
'args' suggest a method call (and more then one value) rather then a single element in an array.

Reply:
Renamed to `arg` to reference just one. Do you feel it is enough?

Reply:
Personally I would prefer 'element' since it relates to the collection which is called 'elements' and 'arg' still sounds like a method/function parameter, but I suppose 'arg' can be acceptable.

Reply:
I like element! Nice suggestion, I will change for that